### PR TITLE
Add Rackspace Metrics product

### PR DESCRIPTION
**Pull request by DonNateR**
PR made for the addition of Rackspace Metrics. Below are a list of updates:

-Remove four articles from "Cloud Monitoring". All of these articles are Metrics specific. 
-Edit metadata of articles to point to Rackspace Metrics
-Add `rackspace-metrics.md` for master product page.
-Fix format of Rackspace Metrics FAQ page to fit FAQ format.
-Add "Rackspace Metrics" to `index.html`.
-Add "Rackspace Metrics to `product.csv`.

Before this PR can be merged, I need to make changes to two more repos.

### CODE REVIEW

**Comment by meker12** [comment]
Hi Nate -- Looks good so far.  I believe that you can merge this PR.  The content just won't go anywhere until you make the other updates.

File: content/index.html [hunk 1]
```diff
@@ -106,6 +106,10 @@
 <div class="twleve primary columns icon intelligence">
 <h4><a href="/how-to/rackspace-intelligence/"><span>Rackspace</span> Intelligence</a></h4>
 </div>
+<div class="three columns icon metrics">
+<h4><a href="/how-to/rackspace-metrics"><span>Rackspace</space> Metrics</a></h4>
```
Comment (meker12) [inline content/index.html]: </space>  should be </span>

File: content/rackspace-metrics/create-a-grafana-dashboard-for-rackspace-metrics.md [hunk 2]
```diff
@@ -173,9 +171,9 @@ We want you to have access to your metrics so you can make informed
 decisions about your infrastructure. Our monitoring service is no cost,
 and so are the metrics that we store.
 
-Q: Should I use Rackspace Intelligence or Grafana?
+### Should I use Rackspace Intelligence or Grafana?
```
Comment (meker12) [inline content/rackspace-metrics/create-a-grafana-dashboard-for-rackspace-metrics.md]: Add another # on "Should I use Rackspace Intelligence..."

File: content/rackspace-metrics/rackspace-metrics.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+### Introduction
+
+- [Rackspace Metrics overview](/how-to/rackspace-metrics-overview)
+
+### General information
+
+- [Create a Grafana dashboard for Rackspace Metrics](/how-to/create-a-grafana-dashboard-for-rackspace-metrics)
+- [Deploy Rackspace Metrics](/how-to/deploying-rackspace-metrics)
+
```
Comment (wolfdancer) [inline content/rackspace-metrics/rackspace-metrics.md]: Please add a link to the EAP section [Early Access Program](https://developer.rackspace.com/docs/metrics/v2/developer-guide/#early-access-program)

**Comment by wolfdancer** [comment]
Thanks for putting this together!  The team will be so happy when we share the link after they are published. 

I put one line comment and below is the additional feedback:

* "deploying-rackspace-metrics.md" file should be "integrate-statsd-with-rackspace-metrics.md", with title changed accordingly
* We need another how-to article "integrate-carbon-forwarder-with-rackspace-metrics.md" based on the content here: https://github.com/rackerlabs/blueflood-carbon-forwarder/blob/master/README.md.  But I would like to see these articles in place and published first if that works.

When this is available, I will submit more PRs to update the Grafana article.  Please watch out for it.

**Comment by DonNateR** [comment]
Merging now.

**Comment by wolfdancer** [comment]
woot!  I see them and reading through them now. 

Two other things that I noticed:
* The main how-to page (where all the products are) looks messed up: https://support.rackspace.com/how-to/
* There should be a "How-to" link from the main documentation site.  I expect that is outside the scope of this PR.  Would you be able to help with adding the link? http://docs.rackspace.com/

**Comment by wolfdancer** [comment]
Here are the additional comments:
* The "Introduction" link on the left points to Rackspace Monitoring.  It should points to the overview article
* "Deploy Rackspace Metrics" article is not fixed.  When opened, the left nav links to Rackspace Monitoring somehow.

**Comment by DonNateR** [comment]
@wolfdancer: I think we've addressed the majority of these issues in other PR's. Let us know if you run into anymore, or keep running into some!

**Comment by wolfdancer** [comment]
@DonNateR have those been merged and deployed? Want to make sure I have the right expectation.  Thanks

**Comment by DonNateR** [comment]
@wolfdancer: Merged and deployed

**Comment by wolfdancer** [comment]
@DonNateR, it looks that the "Deploying Rackspace Metrics" article still need to be fixed in two ways:
* It should be called "Integrate StatsD with Rackspace Metrics"
* The left nav links still points to Rackspace Monitoring.
Thanks.

**Pull request merged**